src/gc.c

```diff
@@ -113,10 +172,31 @@ void GCContext_OnDelete(GCContext* gc) {
 }
 
 void GCContext_ForceInvoke(GCContext* gc, RedisModuleBlockedClient* bc) {
+  if (gc->stopped) return;
+
   BlockClients_push(&gc->bClients, bc);
-  RMUtilTimer_ForceInvoke(gc->timer);
+  GCContext_ForceBGInvoke(gc);
 }
 
 void GCContext_ForceBGInvoke(GCContext* gc) {
-  RMUtilTimer_Signal(gc->timer);
+  RedisModuleCtx* ctx = RSDummyContext;
+  RedisModule_StopTimer(ctx, gc->timerID, NULL);
```

Comment:
You might not have a timer here to stop

Reply:
In that case, it returns RM_ERR but it is legal.